### PR TITLE
New stanza order

### PR DESCRIPTION
**Pull request by vitorgalvao**
Closes https://github.com/caskroom/homebrew-cask/issues/16161.

### CODE REVIEW

**Comment by adityadalal924** [comment]
:+1 LGTM

Will also open up another PR with more style changes to CONTRIBUTING after this one gets merged.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -172,6 +173,63 @@ Additional stanzas you might need for special use-cases:
 
 Even more special-use stanzas are listed at [Optional Stanzas](doc/CASK_LANGUAGE_REFERENCE.md#optional-stanzas) and [Legacy Stanzas](doc/CASK_LANGUAGE_REFERENCE.md#legacy-stanzas).
 
+### Stanza order
+
+Having a common order for stanzas makes Casks easier to update and parse. Empty lines shown here are also important as they help to visually make sense of blocks of information.
```
Comment (adityadalal924) [inline CONTRIBUTING.md]: Alternative wording, not sure how appropriate it is. Should definitely be a line about this being an exhaustive list, not a representative one:

Having a common order for stanzas makes Casks easier to update and parse. Below is the the complete stanza order - not all Casks will have all stanzas. The empty lines shown here are also important, as they help to visually delineate information.

Comment (vitorgalvao) [inline CONTRIBUTING.md]: Updated.

**Comment by adityadalal924** [comment]
Much clearer wording :100:

**Pull request merged**